tests/e2e/composable-indexer/composable-indexer-lib/src/lib.rs

```diff
@@ -1,29 +1,26 @@
 extern crate alloc;
-use fuel_indexer_macros::{graphql_schema, handler};
-use fuel_indexer_schema::Bytes32;
-use fuels_abigen_macro::wasm_abigen;
+use fuel_indexer_macros::indexer;
 
-// IMPORTANT: This must match the namespace in your
-graphql_schema!("composability_test", "schema/schema.graphql");
-wasm_abigen!(
-    no_name,
-    "tests/e2e/composable-indexer/composable-indexer-lib/contracts/ping/out/debug/ping-abi.json"
-);
+#[indexer(
+    abi = "tests/e2e/composable-indexer/composable-indexer-lib/contracts/ping/out/debug/ping-abi.json",
+    namespace = "composability_test",
+    schema = "schema/schema.graphql"
+)]
+mod composability_test {
+    fn function_one(ping: Ping) {
```

Comment:
Last thing (forgot to leave this in the original review).
- So how is block/transaction indexing going to work here?
  - Would we write special handlers for `Block`s and/or `Transaction`s ?
  - For example, what if I wanted access to a `Block` here for indexing (e.g., SwaySwap)
    - Or would this behavior lie somewhere else entirely? 🤔

Reply:
Yea if that's too much for this PR we can always follow up. But will need that before we can start indexing blocks I believe